digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/SevenRestrictionsLimitationValidatorSpec.scala

```diff
@@ -246,101 +246,13 @@ class SevenRestrictionsLimitationValidatorSpec  extends FunSuite with Matchers {
     }
   }
 
-  test("bogieWeightLimit traffic sign without match asset") {
```

Comment:
Deleted tests needs to be investigated

Reply:
There is a test that checks all asset types without match asset related with the all sevenRestrictionsAssets but it was missing the bogieWeightLimit one. So in that test was added the bogieWeightLimit asset type and deleted this test since it was redundant.

```
test("sevenRestiction traffic sign without match asset") {
    sevenRestrictionsAssets.foreach { sevenRestrictionsAsset =>
      massLimitationWithoutMatchedAsset(sevenRestrictionsAsset)
    }
  }
```